src/firefly/js/visualize/ui/ImageSelectPanelState.js

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
This file should probably be call ImageSelectPanelReducer.  It does not hold state it just modifies state.